ReadingList.Api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,49 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ReadingList.Api.Abstractions;
+using ReadingList.Api.QueriesData;
+using ReadingList.Domain.Commands;
+using ReadingList.Domain.Queries;
+
+namespace ReadingList.Api.Controllers
+{
+    [Route("api/[controller]")]
+    public class AccountController : BaseController
+    {
+        [HttpPost]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody] LoginData loginData)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+            
+            var result = await AskAsync(new LoginUserQuery(loginData.Email, loginData.Password));
+
+            if (!result.IsSucceed)
```

Comment:
Code was repeated several times so can be moved into separate method so will look like (just an example):
```cs
return AskResult(result);
```